packages/vscode-lit-xml/src/index.ts

```diff
@@ -0,0 +1,2 @@
+// eslint-disable-next-line @typescript-eslint/no-empty-function
+export function activate() {}
```

Comment:
Why is this fn needed?

Reply:
It's the entrypoint of the extension. Since it has no code functionality, there doesn't need to be anything in the function

Reply:
Could you add a line of comment in the fn body? That way you also don't need to ignore the eslint rule. Empty fn's are fine, as long as they have a comment explaining why they are empty.

Reply:
Done!